Cooper/ClientApp/src.v2/app/services/session/session.service.ts

```diff
@@ -32,6 +36,28 @@ export class SessionService {
     return this.sessionProfile.id;
   }
 
+  public GetSessionUser(): User {
+    const user = new User();
+    user.id = this.sessionProfile.id;
+    user.email = this.sessionProfile.email;
+    user.description = this.sessionProfile.description;
+    user.name = this.sessionProfile.name;
+    user.nickname = this.sessionProfile.nickname;
+    user.photoURL = this.sessionProfile.photoURL;
+
+    return user;
+  }
```

Comment:
it looks strange. it should be other model, or we need converter, or we need constructor or type casting